src/winetricks

```diff
@@ -14953,6 +14953,45 @@ load_openwatcom()
 
 #----------------------------------------------------------------
 
+w_metadata powershell apps \
+    title="Windows Powershell" \
+    publisher="Microsoft" \
+    year="2019" \
+    media="download" \
+    file1="PowerShell-6.1.6-win-x86.zip" \
+    installed_file1="$W_SYSTEM32_DLLS_WIN/WindowsPowerShell/v1.0/powershell.exe"
+
+load_powershell()
+{
+    if test "$W_ARCH" = "win64"; then
+        w_die "Only implemented for 32bit arch at the moment"
+    fi
```

Comment:
You can use w_package_unsupported_win64 for that.

Reply:
I can technically add win64 support if you would prefer that. I don't really see an issue adding it.
However if you would like that indeed - what would be the best example of handling separate x86/64 archives?

Reply:
That would be preferable, but not required.

There are several examples, e.g., amstream:
https://github.com/Winetricks/winetricks/blob/master/src/winetricks#L5586

you'll want to unconditionally install win32 (as win64 always supports win32), then selectively install win64 if $W_ARCH = win64.

Reply:
I am honestly not sure what is the right way to install this properly and integrate into system and how PATH is supposed to be set up in the right way if both 32 and 64 bit things are installed so I guess I'll just leave it as is here